docs-site/website/versioned_docs/version-2.0.0/onnx.md

```diff
@@ -1,7 +1,6 @@
 ---
-id: version-2.0.0-onnx
+id: onnx
 title: ONNX
-original_id: onnx
```

Comment:
@joddiy same here, for versioned_docs, you need to keep the `original_id`

Reply:
Hi, @FeynmanDNA , sorry for late, just fix it.